ChangeLog

```diff
@@ -0,0 +1,237 @@
+version <unreleased>
```

Comment:
Isn't it easier to just prepend the actual string during release rather than remember to always keep `version <unreleased>` on top? Also for substitution pattern we usually use `@UPPERCASE@`.